openshift/init_tenant.go

```diff
@@ -136,6 +140,49 @@ func do(config Config, callback Callback, username, usertoken string, templateVa
 	return nil
 }
 
+// loadTemplate will load the template for a specific version from maven central or from the template directory
+// or default to the OOTB template included
+func loadTemplate(config Config, name string) ([]byte, error) {
+	teamVersion := config.TeamVersion
+	if len(teamVersion) > 0 {
+		url := ""
+		switch name {
+		case "fabric8-online-team-openshift.yml":
+			url = "http://central.maven.org/maven2/io/fabric8/online/packages/fabric8-online-team/$TEAM_VERSION/fabric8-online-team-$TEAM_VERSION-openshift.yml"
+		case "fabric8-online-jenkins-openshift.yml":
+			url = "http://central.maven.org/maven2/io/fabric8/online/packages/fabric8-online-jenkins/$TEAM_VERSION/fabric8-online-jenkins-$TEAM_VERSION-openshift.yml"
+		case "fabric8-online-che-openshift.yml":
+			url = "http://central.maven.org/maven2/io/fabric8/online/packages/fabric8-online-che/$TEAM_VERSION/fabric8-online-che-$TEAM_VERSION-openshift.yml"
+		}
+		if len(url) > 0 {
+			url = strings.Replace(url, "$TEAM_VERSION", teamVersion, -1)
+			if config.Verbose {
```

Comment:
yeah, happy to do that too. I was more thinking that mostly folks would not wanna see that stuff when used in the REST service; but would from the CLI - so it was more a way for the CLI to enable more logging rather than tinkering with debug levels?